### ISSUE TITLE
Installation hangs on wrong VB version

### ISSUE THREAD

**Issue by chrowe** [kalamuna/kalaboxv1#74]
Following instructions at https://github.com/kalamuna/kalabox/wiki/Getting-Started-With-Kalabox
1. Downloaded kalabox-1.0-beta7.dmg
2. Double click the installer
3. Clicked "Box me"
4. Don't remember a terms of service page
5. Entered my password (Luckily I think it was the right one #73 )
6. Then I see this and no further progress
   ![kalabox-hang](https://cloud.githubusercontent.com/assets/240115/3603058/3f94f382-0d13-11e4-8939-2e8094c6fbeb.png)
7. open ~/.kalabox/kalabox.log

```
2014-07-16T17:44:34.988Z - info: Box initialized.
2014-07-16T17:44:34.989Z - info: Kalabox Version: 1.0-beta6
2014-07-16T17:44:34.989Z - info: Mac OS Version: 10.9.3
2014-07-16T17:44:34.990Z - info: VirtualBox Version: 4.3.10r93012
2014-07-16T17:44:34.990Z - info: Vagrant Version: Vagrant 1.6.0
2014-07-16T17:44:37.050Z - info: Existing VirtualBox installation, version 4.3.10
2014-07-16T17:44:37.051Z - info: Existing Vagrant installation, version 1.6.0
2014-07-16T17:44:37.052Z - info: VirtualBox not installed or wrong version. Installing.
```

**Comment by pirog** [comment]
Removed the TOS since we are MIT license aka open source now. 

Did you move the app from the DMG into your applications folder? What's the over/under on whether you entered the correct admin password?

We currently do not support vagrant 1.5+ and probably won't for V1 although there is an open issue for it. Did you try uninstalling with the uninstall.tool option 2 and then reinstalling?

**Comment by chrowe** [comment]
I missed the move to Applications folder step. I tried to [Clarify](https://github.com/kalamuna/kalabox/wiki/Getting-Started-With-Kalabox/_compare/08ed44176ea401f1e34012553fae80c0c1301fbc...f6b850387ff31d5e96b5e7b4c1209d3431922615)  that in the docs. 

So Vagrant 1.2.2 through 1.4.3 are supported?
What about VirtualBox? Is it only 4.2.8 or 4.2.8+

**Comment by chrowe** [comment]
After moving to the Applications folder and installing the correct versions of VirtualBox and Vagrant I still get the same problem.

I have even tried with
- VirtualBox Version: 4.2.8r83876
- Vagrant 1.4.3
- Vagrant version 1.2.2

**Comment by pirog** [comment]
Run the uninstall.tool that comes with the dmg. Choose option #2 and then
try the installer. I would export any vb boxes you have first just in case.

On Wednesday, July 16, 2014, Chris Rowe notifications@github.com wrote:

> After moving to the Applications folder and installing the correct
> versions of VirtualBox and Vagrant I still get the same problem.
> 
> I have even tried with
> - VirtualBox Version: 4.2.8r83876
> - Vagrant 1.4.3
> - Vagrant version 1.2.2
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/kalamuna/kalabox/issues/74#issuecomment-49258560.

## 

Cheers,

Mike Pirog
Kalamuna
www.kalamuna.com

**Comment by chrowe** [comment]
That worked. Thanks

**Comment by pirog** [comment]
Great!

On Thu, Jul 17, 2014 at 7:25 AM, Chris Rowe notifications@github.com
wrote:

> Closed #74 https://github.com/kalamuna/kalabox/issues/74.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/kalamuna/kalabox/issues/74#event-142668077.

## 

Cheers,

Mike Pirog
Kalamuna
www.kalamuna.com
